Persephone/MPDClient/MPDClient.swift

```diff
@@ -119,6 +120,14 @@ class MPDClient {
     idle()
   }
 
+  func seekCurrentSong(timeInSeconds: Float) {
+    noIdle()
+    commandQueue.async { [unowned self] in
+      mpd_run_seek_current(self.connection, timeInSeconds, false)
+    }
+    idle()
```

Comment:
Does this `idle()` rely on the seek command being completed/started? Should it be invoked from `commandQueue`, or should the seek command be run with `sync` instead of `async`?

Reply:
The `idle()` method enqueues the idle in the same serial queue so will not happen until the seek has completed.